<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>AccountRemove</class>
  <widget class="QDialog" name="AccountRemove" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>429</width>
        <height>173</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Remove Account</string>
    </property>
    <layout class="QVBoxLayout" >
      <property name="margin" >
        <number>11</number>
      </property>
      <property name="spacing" >
        <number>6</number>
      </property>
      <item>
        <widget class="QGroupBox" name="gb_account" >
          <property name="title" >
            <string>Remove Account</string>
          </property>
          <layout class="QVBoxLayout" >
            <property name="margin" >
              <number>11</number>
            </property>
            <property name="spacing" >
              <number>6</number>
            </property>
            <item>
              <widget class="QRadioButton" name="rb_remove" >
                <property name="text" >
                  <string>Remove account from Psi only.</string>
                </property>
              </widget>
            </item>
            <item>
              <widget class="QRadioButton" name="rb_removeAndUnreg" >
                <property name="text" >
                  <string>Remove account and try to unregister it from the server.</string>
                </property>
              </widget>
            </item>
            <item>
              <layout class="QHBoxLayout" >
                <property name="margin" >
                  <number>0</number>
                </property>
                <property name="spacing" >
                  <number>6</number>
                </property>
                <item>
                  <widget class="QLabel" name="lb_pass" >
                    <property name="text" >
                      <string>Password:</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="le_pass" >
                    <property name="maximumSize" >
                      <size>
                        <width>100</width>
                        <height>32767</height>
                      </size>
                    </property>
                    <property name="echoMode" >
                      <enum>QLineEdit::Password</enum>
                    </property>
                  </widget>
                </item>
                <item>
                  <spacer name="Spacer2" >
                    <property name="sizeHint" >
                      <size>
                        <width>30</width>
                        <height>0</height>
                      </size>
                    </property>
                    <property name="sizeType" >
                      <enum>Expanding</enum>
                    </property>
                    <property name="orientation" >
                      <enum>Horizontal</enum>
                    </property>
                  </spacer>
                </item>
              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="Spacer3" >
          <property name="sizeHint" >
            <size>
              <width>16</width>
              <height>16</height>
            </size>
          </property>
          <property name="sizeType" >
            <enum>Expanding</enum>
          </property>
          <property name="orientation" >
            <enum>Vertical</enum>
          </property>
        </spacer>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <widget class="BusyWidget" name="busy" />
          </item>
          <item>
            <spacer name="Spacer16" >
              <property name="sizeHint" >
                <size>
                  <width>179</width>
                  <height>16</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <layout class="QHBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="IconButton" name="pb_close" >
                  <property name="text" >
                    <string>&amp;Close</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="IconButton" name="pb_remove" >
                  <property name="text" >
                    <string>&amp;Remove</string>
                  </property>
                  <property name="psiIconName" stdset="0" >
                    <string>psi/remove</string>
                  </property>
                </widget>
              </item>
            </layout>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
</ui>
